test/com/sun/jna/NativeTest.java

```diff
@@ -26,6 +26,7 @@
 import java.io.File;
 import java.lang.reflect.Method;
 import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
```

Comment:
It would be good if the source could be kept java 6 compatible (we will need to switch to minimum build JDK 8 for JDK 10 /11 compatibility). For a test this might be acceptable, but the alternative `Charset.forName("UTF-8")` works.